pkg/kubelet/kubelet.go

```diff
@@ -730,7 +728,7 @@ func NewMainKubelet(kubeCfg *kubeletconfiginternal.KubeletConfiguration,
 			names = []string{klet.GetHostname(), hostnameOverride}
 		}
 
-		klet.serverCertificateManager, err = kubeletcertificate.NewKubeletServerCertificateManager(klet.kubeClient, kubeCfg, klet.nodeName, ips, names, certDirectory)
+		klet.serverCertificateManager, err = kubeletcertificate.NewKubeletServerCertificateManager(klet.kubeClient, kubeCfg, klet.nodeName, ips, names, kubeCfg.CertDirectory)
```

Comment:
since kubeCfg is already passed in here, can you eliminate the final argument?

Reply:
ok, I will do this when the above discussion is confirmed.